### PR TITLE
Add Integration tests for getSubscriptionThrottlingPolicies endpoint

### PR DESCRIPTION
**Pull request by AnuGayan**
This PR will add test cases for **getSubscriptionThrottlingPolicies** endpoint

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/graphqlqueryanalysis/GraphQLQueryAnalysisTest.java [hunk 1]
```diff
@@ -11,9 +11,11 @@
 import org.testng.annotations.*;
 import org.wso2.am.integration.clients.admin.ApiResponse;
 import org.wso2.am.integration.clients.admin.api.dto.*;
+import org.wso2.am.integration.clients.admin.api.dto.RequestCountLimitDTO;
+import org.wso2.am.integration.clients.admin.api.dto.ThrottleLimitDTO;
 import org.wso2.am.integration.clients.publisher.api.v1.dto.*;
 import org.wso2.am.integration.clients.store.api.v1.dto.SubscriptionDTO;
-import org.wso2.am.integration.clients.store.api.v1.dto.ApplicationDTO;
+import org.wso2.am .integration.clients.store.api.v1.dto.ApplicationDTO;
```
Comment (chamilaadhi) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/graphqlqueryanalysis/GraphQLQueryAnalysisTest.java:18]: Does it get compiled with a space here 🤔

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/publisher/APIMGetAllSubscriptionThrottlingPolicies.java [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            (TestUserMode userMode) {
+        this.userMode = userMode;
+    }
+
+    @DataProvider
+    public static Object[][] userModeDataProvider() {
+        return new Object[][]{
+                new Object[]{TestUserMode.SUPER_TENANT_ADMIN},
+//                new Object[]{TestUserMode.TENANT_ADMIN},
+        };
```
Comment (chamilaadhi) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/publisher/APIMGetAllSubscriptionThrottlingPolicies.java:50]: Any reason for only running for supertenant?. Shall we enable for tenants as well?

Comment (AnuGayan) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/publisher/APIMGetAllSubscriptionThrottlingPolicies.java:50]: This was disabled in APIM634GetAllTheThrottlingTiersFromThePublisherRestAPITestCase.I followed the same approach.

**Comment by chamilaadhi** [comment]
merged https://github.com/wso2/carbon-apimgt/pull/9971. update the carbon.apimgt version ones the release is done

**Review by chamilaadhi — APPROVED** [review]

**Pull request merged**